packages/cli/package.json

```diff
@@ -127,7 +127,7 @@
     "stack-trace": "0.0.10",
     "style-loader": "^2.0.0",
     "terser-webpack-plugin": "^4.2.3",
-    "typescript": "^4.2.4",
+    "typescript": "~4.2.4",
```

Comment:
This is completely unrelated so I could move into a separate PR if desired.

From what I understand, Yarn needs to manually patch some dependencies like TS on every release. TS 4.5 just came out, which apparently caused some issues in the PnP pipeline.

Shouldn't be an issue to just restrict this to a patch/TS minor

Reply:
eh, if it makes CI pass leave it in.